includes/admin/settings/display-settings.php

```diff
@@ -21,12 +21,81 @@ function edd_admin_header() {
 	if ( ! edd_is_admin_page( '', '', false ) ) {
 		return;
 	}
-
 	$numberNotifications = EDD()->notifications->countActiveNotifications();
+	$current_page        = ! empty( $_GET['page'] ) ? $_GET['page'] : '';
+	$is_single_view      = ! empty( $_GET['view'] ) ? true : false;
+
+	$page_title = '';
+	switch( $current_page ) {
+		case 'edd-settings':
+			$page_title = __( 'Settings', 'easy-digital-downloads' );
+			break;
+		case 'edd-reports':
+			$page_title = __( 'Reports', 'easy-digital-downloads' );
+			break;
+		case 'edd-payment-history':
+			$page_title = __( 'Orders', 'easy-digital-downloads' );
+			break;
+		case 'edd-discounts':
+			$page_title = __( 'Discounts', 'easy-digital-downloads' );
+			break;
+		case 'edd-customers':
+			$page_title = __( 'Customers', 'easy-digital-downloads' );
+			break;
+		case 'edd-tools':
+			$page_title = __( 'Tools', 'easy-digital-downloads' );
+			break;
+		default:
+			if ( ! empty( $_GET['page'] ) ) {
+				$page_title = ucfirst( str_replace( 'edd-', '', $current_page ) );
+			} else {
+				$page_title = __( 'Downloads', 'easy-digital-downloads' );
+			}
+
+			break;
+	}
+
+	$page_title = apply_filters( 'edd_settings_page_title', $page_title, $current_page );
+	if ( ! empty( $page_title ) && empty( $is_single_view ) ) {
+		?>
+		<style>
+			.wrap > h1,
+			.wrap h1.wp-heading-inline,
+			.wrap h2,
+			.page-title-action {
+				display: none;
```

Comment:
This removes the heading from the screen, which is an accessibility issue. We intentionally un-hid all of the headings in a previous issue/PR. Is there another way we can accomplish this?

Reply:
This makes all the 'section' labels an H1: https://github.com/awesomemotive/easy-digital-downloads/pull/9244/commits/aec57702a9a3d63c12cd01a96b0f25decc19f46d